packages/server/tests/clients/relayClient.ts

```diff
@@ -59,7 +59,6 @@ export default class RelayClient {
         try {
             const res = await this.call(methodName, params, requestId);
             this.logger.trace(`${requestIdPrefix} [POST] to relay '${methodName}' with params [${params}] returned ${JSON.stringify(res)}`);
-            Assertions.expectedError();
```

Comment:
By removing this you're implying the method succeeds when it shouldn't no?

If the test is getting here doesn't that mean the call was not failing?
In which case the test is operating correctly and failing then the expectation is not met i.e. there's no revert error.

Reply:
Agreed.  Upon a second look I see that the test is not asserting the expected JsonRpcError.  I'll close this PR.